src/System Application/App/Entity Text/src/EntityTextImpl.Codeunit.al

```diff
@@ -63,6 +79,9 @@ codeunit 2012 "Entity Text Impl."
     var
         EntityText: Record "Entity Text";
     begin
+        if SourceScenario = Enum::"Entity Text Scenario"::"Reminder Text" then
```

Comment:
Same comment as before, this is adding a dependency in a "generic" tool to a feature, is this something we really want to do? Wouldn't it be better to leave this code agnostic to the feature that use it and write the code for reminders on top of it?

Reply:
Same answer as before - it is a conscious decision not to raise events